prepare-release.sh

```diff
@@ -10,6 +10,8 @@ rm -r node_modules
 rm package-lock.json
 npm install
 npm ci
+# Edit default Arc version in migrate.js file
+read -n 1 -s -r -p $'Please edit default Arc version in migrate.js file to the latest version.\nPress any key to continue release.\n'
```

Comment:
why is that needed ? the version can be fetched from latest

Reply:
I initially thought about using latest but this can cause issues when new Arc is released with changes not yet supported by the migration.